plugins/obs-filters/cmake/rnnoise.cmake

```diff
@@ -1,46 +1,48 @@
 option(ENABLE_RNNOISE "Enable building with RNNoise noise supression filter" ON)
 
 if(ENABLE_RNNOISE)
-  if(OS_WINDOWS OR OS_MACOS)
-    find_package(Librnnoise REQUIRED)
+  if(NOT OS_WINDOWS AND NOT OS_MACOS)
+    add_library(obs-rnnoise OBJECT)
```

Comment:
)RNNoise can be provided as a package/module, CMake 3.0 should support system RNNoise install.

Reply:
Always try the system install first even if Ubuntu does not.

Reply:
Got it - so simply try to find the package but make it not required and fall back on the vendored sourced if the target is not found.